src/test/java/nl/tudelft/pixelperfect/gamestates/PlayStateTest.java

```diff
@@ -60,7 +62,7 @@ public void testUpdate() {
     verify(mockScheduler).update(1f);
     verify(mockSpaceship).update(anyFloat());
 
-    if (Constants.isDebug) {
+    if (Settings.getIsDebug()) {
       verify(mockHeadsUp).updateHud();
     }
```

Comment:
I don't think it's allowed to have if statements in test cases, as shown in the lectures from SIG